nano/node/active_transactions.cpp

```diff
@@ -363,74 +274,17 @@ void nano::active_transactions::request_confirm (nano::unique_lock<std::mutex> &
 		// Broadcast and request confirmation
 		else if (election_l->skip_delay || election_l->election_start < cutoff_l)
 		{
-			bool increment_counter_l{ true };
+			solicitor.add (election_l);
 			// Escalate long election after a certain time and number of requests performed
 			if (election_l->confirmation_request_count > 4 && election_l->election_start < long_election_cutoff_l)
 			{
 				election_escalate (election_l, transaction_l, roots_size_l);
 			}
-			// Block broadcasting
-			if (election_l->confirmation_request_count % 8 == 1)
-			{
-				election_broadcast (election_l, transaction_l, blocks_bundle_l, inactive_l, root_l);
-			}
-			// Confirmation requesting
-			if (election_l->confirmation_request_count % 4 == 0)
-			{
-				// If failed to insert into any of the bundles (capped), don't increment the counter so that the same root is sent for confirmation in the next loop
-				if (!election_request_confirm (election_l, representatives_l, roots_size_l, single_confirm_req_bundle_l, batched_confirm_req_bundle_l))
-				{
-					increment_counter_l = false;
-				}
-			}
-			if (increment_counter_l || node.network_params.network.is_test_network ())
-			{
-				++election_l->confirmation_request_count;
-			}
 		}
 	}
 	ongoing_broadcasts = !blocks_bundle_l.empty () + !batched_confirm_req_bundle_l.empty () + !single_confirm_req_bundle_l.empty ();
```

Comment:
This is not getting decremented anymore. With the change to start broadcasting / sending to channels within the solicitor, it seems this might not be necessary. The request loop waits on these here: https://github.com/nanocurrency/nano-node/blob/42e5a27545e73606993dcfc520ea8b69d6e75e0f/nano/node/active_transactions.cpp#L324-L327

Reply:
Any concern if requests pile up in slow nodes, with more getting added before the previous ones go through?